core/src/avm1/object/stage_object.rs

```diff
@@ -118,27 +118,46 @@ impl<'gc> StageObject<'gc> {
         context: &mut UpdateContext<'_, 'gc, '_>,
         case_sensitive: bool,
     ) -> Option<Value<'gc>> {
-        if let Some(slice) = name.slice(..6) {
-            let level_prefix = WStr::from_units(b"_level");
+        if let Some(prefix) = name.slice(..6) {
             let is_level = if case_sensitive {
-                slice == level_prefix
+                prefix == WStr::from_units(b"_level") || prefix == WStr::from_units(b"_flash")
```

Comment:
can we just use `prefix == b"_level"` in comparisons? I see this done with `AvmString` at least, not sure about this slice

Reply:
You're right; I applied the change and rebased.